### ISSUE TITLE
The test fails in  pypy stage due to 'typed-ast.'

### ISSUE THREAD

**Issue by nozzy123nozzy** [PyCQA/pylint#5190]
### Bug description

The 'tox -r -e pypy' always fails.

That command causes the following error,
ERROR: could not install deps [-r/home/.../pylint/requirements_test.txt]; v = InvocationError('/home/.../pylint/.tox/pypy/bin/python -m pip install -r/home/.../pylint/requirements_test.txt', 1) 

To pass tests, we should remove the requirements for 'pre-commit'  from requirements.txt. Because the root cause is that 'typed-ast,' which black depends, doesn't support pypy at this moment and future.
(See. Incompatibility section of 
  https://github.com/python/typed_ast
) 
I show the patch to fix this problem as below,
```diff
diff --git a/requirements_test.txt b/requirements_test.txt
index fce9ec8d..504b1122 100644
--- a/requirements_test.txt                                                                                                                            +++ b/requirements_test.txt
@@ -1,4 +1,3 @@
--r requirements_test_pre_commit.txt
 -r requirements_test_min.txt
 coveralls~=3.2
 coverage~=5.5
```

I've confirmed this patch resolves this problem, and 'tox -r'  passes all tests.

Could someone apply this patch, please?
Many thanks for considering my request.

### Command used

```shell
# To reproducing  this problem 
pypy is installed by 'pyenv install pypy3.6-7.3.3'
'pylint/.tox/pypy/bin/python -V' shows,
Python 3.6.12 (db1e853f94de, Nov 18 2020, 09:49:19)
[PyPy 7.3.3 with GCC 7.3.1 20180303 (Red Hat 7.3.1-5)]  
'tox --version' shows,
3.21.4 imported from /usr/lib/python3/dist-packages/tox/__init__.py

And then,

git clone https://github.com/PyCQA/pylint.git
cd pylint
tox -r pypy

Finally test in pypy stage is failed.
```

### Pylint output

```shell
'tox -r -e formatting' causes nothing to problem.
```

### Expected behavior

Expect result is,

tox -r -e pypy
..snip...
  pypy: commands succeeded
  congratulations :)

### Pylint version

```shell
pylint (git commit 80205dc813e17fc416e56bace982c971a10553fb)
astroid-2.8.3
tox 3.21.4
pypy3.6-7.3.3
```

### OS / Environment

debian sid

### Additional dependencies

astroid==2.8.3
attrs==19.3.0
backports.entry-points-selectable==1.1.0
certifi==2021.10.8
cffi==1.14.3
cfgv==3.3.1
charset-normalizer==2.0.7
cli-ui==0.10.3
colorama==0.4.4
contextlib2==21.6.0
coverage==5.5
coveralls==3.2.0
distlib==0.3.3
docopt==0.6.2
execnet==1.9.0
filelock==3.3.1
gprof2dot==2021.2.21
greenlet==0.4.13
identify==2.3.0
idna==3.3
importlib-metadata==4.8.1
importlib-resources==5.3.0
iniconfig==1.1.1
isort==5.9.3
lazy-object-proxy==1.6.0
mccabe==0.6.1
nodeenv==1.6.0
packaging==21.0
platformdirs==2.4.0
pluggy==1.0.0
pre-commit==2.15.0
py==1.10.0
py-cpuinfo==8.0.0
pyenchant==3.2.2
pylint @ file:///home/.../pylint/.tox/.tmp/package/1/pylint-2.11.2.dev0.zip
pyparsing==2.4.7
pytest==6.2.5
pytest-benchmark==3.4.1
pytest-cov==3.0.0
pytest-forked==1.3.0
pytest-profiling==1.7.0
pytest-xdist==2.4.0
PyYAML==6.0
readline==6.2.4.1
requests==2.26.0
schema==0.7.4
six==1.16.0
tabulate==0.8.9
tbump==6.5.0
toml==0.10.2
tomlkit==0.7.2
types-pkg-resources==0.1.3
types-toml==0.10.1
typing-extensions==3.10.0.2
Unidecode==1.3.2
urllib3==1.26.7
virtualenv==20.8.1
wrapt==1.13.2
zipp==3.6.0
